src/tasks/Crossgen2Tasks/RunReadyToRunCompiler.cs

```diff
@@ -346,7 +348,7 @@ private string GenerateCrossgen2ResponseFile()
 
             if (!string.IsNullOrEmpty(Crossgen2ExtraCommandLineArgs))
             {
-                foreach (string extraArg in Crossgen2ExtraCommandLineArgs.Split(new char[]{';'}, StringSplitOptions.RemoveEmptyEntries))
+                foreach (string extraArg in Crossgen2ExtraCommandLineArgs.Split(s_separator, StringSplitOptions.RemoveEmptyEntries))
```

Comment:
What .NET version does this component target?  If it's only .NET Core, it can instead just use the Split overload that takes a single char rather than using a cached array.